util/chplenv/chpl_llvm.py

```diff
@@ -372,18 +374,26 @@ def get_system_llvm_clang(lang):
         clang = os.path.join(bindir, clang_name)
 
         if not os.path.exists(clang):
-            # also try /usr/bin/clang since some OSes use that
-            # for the clang package
+            # try /usr/bin/clang-<version> or /usr/bin/clang
+            # since some OSes use that for the clang package
+            paths = [ ]
+
             usr_bin = "/usr/bin"
-            clang2 = os.path.join(usr_bin, clang_name);
-            if os.path.exists(clang2):
-                llvm_config = find_system_llvm_config()
-                # check that clang --version matches llvm-config --version
-                clangv = run_command([clang2, '--version']).strip()
-                llvmv = run_command([llvm_config, '--version']).strip()
-
-                if llvmv in clangv:
-                    clang = clang2
+            llvm_config = find_system_llvm_config()
+            llvm_version, ignored_err = check_llvm_config(llvm_config)
```

Comment:
Given that this is in `get_system_llvm_clang` I'm assuming this won't execute for `CHPL_LLVM=bundled` (in which case we'd have to make sure that we've built `llvm-config` first). I guess if that's an issue the old code would have had it too though.

Reply:
Yes, this does not execute for `CHPL_LLVM=bundled`.